src/pages/sign-up/SignUp.tsx

```diff
@@ -51,15 +51,16 @@ const SignUp = () => {
         .then((response) => {
           if (response.data) {
             dispatch(autoPopulateEmailAction(data.email))
+            console.log(response)
+            navigate('/signin')
+          } else {
+            navigate('/signin', { state: { accountSvcError: true } })
           }
-          //TODO
-          console.log(response)
         })
         .catch((error) => {
-          //TODO
           console.log(error)
+          navigate('/signin')
```

Comment:
After doing the review comments changes for account service, you can display some general error messages for the exceptions also.